src/lib/components/drs/pages/DrsObjectForm.js

```diff
@@ -0,0 +1,225 @@
+    const [error, setError] = useState(null);
+
+    const applyReadOnly = () => {
+        let nonEditableOnEditForm = ['id', 'isBundle'];
+        let p = {};
+        Object.keys(props.groupedFormProps).forEach(key => {
+            switch (props.formViewType) {
+                case FormViewType.SHOW:
+                    p[key] = {...props.groupedFormProps[key], readOnly:true};
+                    break;
+                case FormViewType.NEW:
+                    p[key] = {...props.groupedFormProps[key], readOnly:false};
+                    break;
+                case FormViewType.EDIT:
+                    // TODO HANDLE EDIT
+                    break;
+            }
+        })
+        return p;
+    }
+
+    // visualize an editable component if on NEW or EDIT page
+    // visualize only VIEW page ONLY if the property is non-null/non-empty
+    const vis = value => {
+        if (props.formViewType === FormViewType.NEW || props.formViewType === FormViewType.EDIT) {
+            return true;
+        }
+
+        switch (typeof value) {
+            case 'string':
+                return value !== '';
+                break;
+            case 'number':
+                return true;
+                break;
+            case 'object':
+                if (value instanceof Array) {
+                    return value.length > 0;
+                }
+        }
+        return false;
+    }
+
+    const p = applyReadOnly();
+
+    return (
+      <div>
+        <meta
+            name="viewport"
+            content="minimum-scale=1, initial-scale=1, width=device-width"
+        />
+        <Container maxWidth='lg'>
+            <Snackbar
+                anchorOrigin={{ vertical: 'bottom', horizontal: 'center'}}
+                open={error}
+                onClose={() => setError(null)}
+            >
+                <Alert onClose={() => setError(null)} severity="error" variant="filled">
+                    {error ? `Error: ${error.message}` : null }
+                </Alert>
+            </Snackbar>
+
+            <Grid container justify='space-between' alignItems='center'>
+                <Grid item xs={2} align='left'>
+                    <Button variant='contained' component={Link} to='/drs' color='primary' size='large'>
+                        <Typography variant='button'>DRS Index</Typography>
+                    </Button>
+                </Grid>
+                <Grid item xs={8}>
+                    <Typography align='center' variant="h3" gutterBottom>{props.title}</Typography>
+                </Grid>
+                <Grid item xs={2} />
+            </Grid>
+            <Box pb={4}>
+                <form>
+                    {vis(p.id.id) ? <Id {...p.id} /> : null}
+                    {vis(p.name.name) ? <Name {...p.name} /> : null}
+                    {vis(p.description.description) ? <Description {...p.description} /> : null }
+                    <Grid container justify='space-evenly' spacing={4}>
```

Comment:
```suggestion
                    <Grid container justify='flex-start' spacing={4}>
```
Currently, if the Version field is not populated, the Created Time and Updated Time fields are aligned in the centre of the screen which is not consistent with the rest of the form. If justify='flex-start' is used, then the fields will be left aligned, regardless of whether Version is populated.